backend/contact/database/datatypes_test.go

```diff
@@ -59,10 +59,10 @@ const payload = `{
 	}
 }`
 
-func TestJSONB_Scan(t *testing.T) {
+func TestPerson_Scan(t *testing.T) {
```

Comment:
This is just testing the scanner and valuer for a Person so just move the test file in the same folder as the struct